packages/engine-server/src/utils.ts

```diff
@@ -326,4 +328,39 @@ export class EngineUtils {
   static getLocalEngineUrl({ port }: { port: number }) {
     return APIUtils.getLocalEndpoint(port);
   }
+
+  /**
+   * @param param0
+   * @returns
+   */
+  static async refreshNoteLinksAndAnchors({
+    note,
+    engine,
+    notesMap,
+  }: {
+    note: NoteProps;
+    engine: DEngineClient;
+    notesMap: Map<string, NoteProps>;
+  }) {
+    const maxNoteLength = ConfigUtils.getWorkspace(engine.config).maxNoteLength;
+    if (
+      note.body.length <
```

Comment:
Is it possible to return something in case length > max length? At least let's add some note in the function description.

Reply:
added a note.